manifests/v/vim/vim/8.2.3187/vim.vim.installer.yaml

```
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: vim.vim
+PackageVersion: 8.2.3187
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3187/gvim_8.2.3187_x64.exe
+  InstallerSha256: 1C2368FCE9EC4E227CE438F85FC5AF9551F524609B24C81790B973A78388F9FB
+  InstallerLocale: en-US
+  UpgradeBehavior: install
+- Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3187/gvim_8.2.3187_x86.exe
+  InstallerSha256: EA25C0DF9D9D74ED8DE2809E98BD7E2B413C4418AD3206C507A4BCA4833BE55F
```

Signed version just came out.

```suggestion
  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3187/gvim_8.2.3187_x86_signed.exe
  InstallerSha256: 72EC79520F91F1B1F6128F846115F5628692C87DA6EAEEADAFA51DBFDAB09F3D
```

---

Thanks!